product_docs/docs/biganimal/release/getting_started/02_connecting_to_your_cloud/index.mdx

```diff
@@ -25,7 +25,11 @@ Before connecting to your cloud,
 
     - arn:aws:iam::aws:policy/ServiceQuotasFullAccess
 
- - **If connecting to an Azure subscription**, ensure that in [Azure Cloud Shell](https://shell.azure.com/), your environment is running:
+- **If connecting to an Azure subscription, ensure that:** 
+
+   - You are assigned either the Global Administrator role or have BOTH the Privileged Role Administrator and Cloud Application Administrator role assignments in Azure AD.
```

Comment:
```suggestion
   - You are assigned either the Global Administrator role or the Privileged Role Administrator role in Azure AD.
```

Reply:
updating the content based on latest messages in the Slack thread https://edb-starlight.slack.com/archives/C02JZMQMSAV/p1664638497509379?thread_ts=1662471859.174509&cid=C02JZMQMSAV 
Feel free to rephrase if needed